Configuration/Adaptive_Meshing.cfg

```diff
@@ -33,6 +33,10 @@ gcode:
     {% set points_x = (((adapted_x_max - adapted_x_min) / max_probe_point_distance_x) | round(method='ceil') | int) + 1 %}          # Define probe_count's x point count and round up
     {% set points_y = (((adapted_y_max - adapted_y_min) / max_probe_point_distance_y) | round(method='ceil') | int) + 1 %}          # Define probe_count's y point count and round up
 
+    {% set probe_dock_enable = printer["gcode_macro _KAMP_Settings"].probe_dock_enable | abs %}                                     # Is Dock enabled?
+    {% set attach_macro = printer["gcode_macro _KAMP_Settings"].attach_macro | default('Attach_Probe') | string %}                  # Get attach macro
```

Comment:
I don't think that setting a default should be done. More precisely, there should be a default but the default value should be some special keyword (like `SAFETY_SENTINEL`).

Then the macro should check if `probe_dock_enable` is True and if so, ensure that both `attach_macro` and `detach_macro` and **not** equal to `SAFETY_SENTINEL`.

Reply:
That is not my project. See it like reporting an issue. I will not invest more time in it. This fixed it for me and I have thought it would be nice to contribute it.

Reply:
> That is not my project. See it like reporting an issue. I will not invest more time in it. This fixed it for me and I have thought it would be nice to contribute it.

Submitting a PR is not the same as reporting an issue. As the author of the PR, technically, you are responsible for responding to comment if you want your change to be accepted. That's just how Open Source works.

I see that you've closed the PR so hopefully someone else will pick up where you are leaving off.